integrations/doubleclick-floodlight/lib/index.js

```diff
@@ -34,6 +41,18 @@ var Floodlight = (module.exports = integration('DoubleClick Floodlight')
  */
 
 Floodlight.prototype.initialize = function() {
+  // In the initialize method:
+  // Check if we should load the DoubleClick ID pixel (and only proceed if we haven't already done so).
+  if (this.options.getDoubleClickId && this.options.googleNetworkId) {
+    // Load the doubleclick pixel.
+    this.load('doubleclick id', {
+      googleNetworkId: this.options.googleNetworkId,
+      segmentWriteKey: this.options.segmentWriteKey,
+      // TODO: handle userId being nulls/undefined.
```

Comment:
Would be possible to investigate what would happen if userId is null/undefined? It's easy to reproduce, just load this script without an identify call being fired.

Reply:
@CarlosMecha This already tested, i have attached the screenshot where **userID** being passed as null if not present. Do we need more investigation around this ?